shared/domain/src/main/java/gov/va/vro/model/mas/request/MasAutomatedClaimRequest.java

```diff
@@ -0,0 +1,62 @@
+package gov.va.vro.model.mas.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import gov.va.vro.model.mas.ClaimDetail;
+import gov.va.vro.model.mas.VeteranIdentifiers;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Builder
+@Getter
+@Schema(name = "MASClaimDetailsRequest", description = "Initiate a MAS request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MasAutomatedClaimRequest {
```

Comment:
I separated the REST request from the payload, because the payload is cluttered with all sorts of fields needed for processing, which should not be in the request.